README.md

```diff
@@ -0,0 +1,120 @@
+# The YaST Ruby Testing Image
+
+[![Build Status](https://travis-ci.org/yast/ci-cpp-container.svg?branch=master)](https://travis-ci.org/yast/ci-cpp-container)
+
+This git repository contains the configuration used to build the docker
+image used for [TravisCI](https://travis-ci.org/).
+The resulting docker image is available at https://registry.opensuse.org/.
+
+## Automatic Rebuilds
+
+- The image is rebuilt whenever a commit it pushed to the `master` branch.
+- The [yast-ci-cpp-container-master](
+  https://ci.opensuse.org/view/Yast/job/yast-ci-cpp-container-master/)
+  Jenkins job copies the configuration to the [YaST:Head/ci-cpp-container](
+  https://build.opensuse.org/package/show/YaST:Head/ci-cpp-container)
+  OBS project
+- The OBS tracks the dependencies and rebuilds the image if any dependant package
+  is updated.
+
+## Triggering a Rebuild Manually
+
+If for some reason the automatic rebuild do not work or it failed you can
+trigger the rebuild in the OBS just like for the other regular packages.
+
+
+## The Image Content
+
+This image is based on the latest openSUSE Tumbleweed image, additionally
+it contains the packages needed for running the tests for YaST packages written
+in Ruby. It is possible to install additional packagers if needed, see the
```

Comment:
Ruby? It contains ruby packages but that is not the intention of the image.

Reply:
ah, copy&paste from the Ruby image...